tests/postgresql/test-postgresql.cpp

```diff
@@ -689,6 +689,23 @@ TEST_CASE("PostgreSQL ORM cast", "[postgresql][orm]")
     sql << "select :a::int", use(v); // Must not throw an exception!
 }
 
+bool table_name_equals_ignore_schema(const std::string& table_name_with_schema, const std::string& table_name)
```

Comment:
It's a bit surprising to have such a function rather than a simple(r) `get_table_name_without_schema()` and then continuing to use simple string comparison below.

Reply:
I've changed this